pyproject.toml

```diff
@@ -4,4 +4,4 @@ package = "lahja"
 filename = "docs/release_notes.rst"
 directory = "newsfragments"
 underlines = ["-", "~", "^"]
-issue_format = "`#{issue} <https://github.com/ethereum/lahja/issues/{issue}>`__" 
\ No newline at end of file
+issue_format = "`#{issue} <https://github.com/ethereum/lahja/issues/{issue}>`__" 
```

Comment:
This is pointing to a foreign repository (lahja). It needs to point to this repo.